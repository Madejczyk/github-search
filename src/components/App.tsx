import * as React from "react";
import "./app.scss";
import { GithubSearch } from "./github-search/GithubSearch";

export const App = () => <>
    <GithubSearch />
</>;
