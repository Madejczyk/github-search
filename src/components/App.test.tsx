import { mount } from "enzyme";
import * as React from "react";
import { App } from "./App";

test("App should render correctly", () => {
    const app = mount(<App />);
    expect(app.text()).toEqual("Hello world!");
    expect(app).toMatchSnapshot();
});
