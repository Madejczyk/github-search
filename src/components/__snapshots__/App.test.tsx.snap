// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should contains GitHub search 1`] = `
<form
  class="github-search"
>
  <div
    class="title"
  >
    GitHub User details
  </div>
  <input
    class="search-input"
    placeholder="User name"
    type="text"
    value=""
  />
  <input
    class="submit-button"
    disabled=""
    type="submit"
    value="Search"
  />
</form>
`;
